db/migrations/20190717104034-create-alter-articles-table.js

```diff
@@ -2,6 +2,12 @@ module.exports = {
   up: (queryInterface, Sequelize) => queryInterface.addColumn('Articles', 'reads', {
     type: Sequelize.INTEGER,
     defaultValue: 0,
+  }).then(() => {
+    queryInterface.addColumn('Articles', 'publish', {
```

Comment:
I think it would be ideal for this nee column to be in a separate migration file... because the migration might not know there is a new migration if you add it with an already existing one.. making it not to run the new migration

Reply:
ok.... that makes sense would implement that.